utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -24,9 +24,8 @@ class UrlImageParser private constructor(
   private val htmlContentTextView: TextView,
   private val entityType: String,
   private val entityId: String,
-  private val imageLoader:ImageLoader
+  @ImageLoaderAnnotation private val imageLoader: ImageLoader
```

Comment:
This class is not up-to-date with develop. Implementation of `UrlImageParser` is changed it doesn't include `ImageLoaderAnnotation`.

Reply:
Ditto: testcases for `UrlImageParser` too.

Reply:
Good catch. I think I merged this part wrong when I was pulling in the newer changes, fixed both (removed the annotation and UrlImageParser sets).